src/els_methods.erl

```diff
@@ -110,6 +111,8 @@ initialize(Params, State) ->
                }
           , definitionProvider => els_definition_provider:is_enabled()
           , referencesProvider => els_references_provider:is_enabled()
+            %% AZ:TODO: different enable flag for documentHighlightProvider
+          , documentHighlightProvider => els_references_provider:is_enabled()
```

Comment:
Why not create a new `els_document_highlight_provider` and move the code added to `els_references_provider` there?

Reply:
I was not sure initially how much overlap there would be, but I presumed that would be the eventual outcome. Will do it now.